components/01-globals/variables/_variables.scss

```diff
@@ -1,23 +1,23 @@
 // Colors
 $white                 : #fff !default;
 $black                 : #000 !default;
-$red                   : #d9534f !default;
-$orange                : #f0ad4e !default;
+$red                   : #eb3d2f !default;
+$orange                : #fab216 !default;
 $yellow                : #ffd500 !default;
-$green                 : #5cb85c !default;
+$green                 : #00b840 !default;
 $blue                  : #0275d8 !default;
 $teal                  : #5bc0de !default;
 $pink                  : #ff5b77 !default;
 $purple                : #613d7c !default;
 
 // Grayscale colors
 $gray-dark             : #292b2c !default;
-$gray                  : #464a4c !default;
-$gray-light            : #636c72 !default;
+$gray                  : #d1d0d4 !default;
+$gray-light            : #f5f5f6 !default;
 $gray-lighter          : #eceeef !default;
 $gray-lightest         : #f7f7f9 !default;
 
-// Smantic color scheme
+// Semantic color scheme
 $color-primary         : $blue !default;
```

Comment:
primary color shouldn't be green? we haven't got so much blue in our design ;)

Reply:
I've updated `$green` variable with our lovely 'alpaca green'. But it leads to another confusion : 
![image](https://user-images.githubusercontent.com/12099511/31654572-c3419eee-b326-11e7-9fb8-5f91aba7fc51.png)
What do you think Anna, is that ok ?

Reply:
If our success color is the same as primary one than it's ok. We will be changing color whith Vasyl later so it's ok :)